### ISSUE TITLE
Not able to start flatpak VSCodium from launcher

### ISSUE THREAD

**Issue by 610th** [pop-os/launcher#78]
When I try to launch flatpak VSCodium using `SUPER` + `/`, the app launches but the app window is completely black. If I start VSCodium from Applications Menu, everything works just fine. I'm using the Nvidia  20.04 version of Pop!_OS.

**Comment by 610th** [comment]
Maybe this is more of a support case than an actual issue. Feel free to remove if so. I experienced the same problem with the snap version of VSCodium. Haven't experimented with many other applications. I'm quite new to Pop.

**Comment by leviport** [comment]
How'd you install VSCodium? The flatpak installed from Pop Shop seems to be working fine for me.

**Comment by 610th** [comment]
Both via Pop Shop and command line. Can't remember what I tried first. Unfortunately, it seems like my Pop Shop has broken. I only get "Abort due to failure" when I try to install something from flathub. Now I have to debug Pop Shop.

I can see this in the syslog

```
Jan  6 21:24:42 pop io.elementary.a[5417]: FlatpakBackend.vala:917: Flatpak installation failed: While pulling app/com.vscodium.codium/x86_64/stable from remote flathub: open(O_TMPFILE): Permission denied (detail: 0)
Jan  6 21:24:42 pop io.elementary.a[5417]: Package.vala:573: Operation failed for package VSCodium - Aborted due to failure
```

**Comment by leviport** [comment]
Might be worth trying a `flatpak repair --user` and seeing if it works after that.

**Comment by 610th** [comment]
Did not help, unfortunately. Strace + deleting locking files and/or upgrading flatpack might be a solution.

**Comment by 610th** [comment]
I managed to fix my flatpak installation by doing ´sudo chmod 1777 /var/tmp´. Found the fix here: https://github.com/flatpak/flatpak/issues/2808 . My guess is that a systemd generator that I was fiddling with broke the permissions.

I re-installed VSCodium via Pop Shop and now I experience the same behavior. When I start via launcher I get a black window and when I start via the applications menu everything works.
